lib/config.rb

```diff
@@ -29,6 +29,10 @@ def builds_dir
       @builds_path ||= File.join($root_path, 'tmp', 'builds')
     end
 
+    def jobs_dir
+      @jobs_path ||= File.join($root_path, 'tmp', 'jobs')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.